core-api/src/main/resources/definitions/exposed-services.json

```diff
@@ -639,6 +639,20 @@
       "apiIncluded": false,
       "visibleForDatalake": false,
       "visibleForDatahub": true
+    },
+    {
+      "name": "KNOX_TOKEN_INTEGRATOR",
+      "displayName": "Token Integration",
+      "serviceName": "KNOX",
+      "knoxService": "KNOX_TOKEN_INTEGRATOR",
+      "knoxUrl": "/tokengen/index.html",
```

Comment:
@doktoric - the `knoxUrl` should be updated to `/homepage/home` too.
We really appreciate your help with this!